lib/rules/web/admin_console/4.4/yaml_page.xyaml

```diff
@@ -0,0 +1,62 @@
+click_yaml_tab:
```

Comment:
If you indeed need to add a new file, maybe use `yaml.xyaml` since not all rules are about yaml page